bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -456,35 +456,35 @@ All these channels are read only.
-|                  | BOOT               | Triggered when a camera is booting                                                                                                                                               |
-|                  | SD                 | Triggered when a camera SD card status was changed                                                                                                                               |
-|                  | ALIM               | Triggered when a power supply status was changed                                                                                                                                 |
-|                  | NEW_MODULE         | Triggered when a new module was discovered                                                                                                                                       |
-|                  | MODULE_CONNECT     | Triggered when a module gets connected                                                                                                                                           |
-|                  | MODULE_DISCONNECT  | Triggered when a module gets disconnected                                                                                                                                        |
-|                  | MODULE_LOW_BATTERY | Triggered when the battery of a module gets low                                                                                                                                  |
-|                  | MODULE_END_UPDATE  | Triggered when a firmware update of a module is done                                                                                                                             |
-|                  | TAG_BIG_MOVE       | Triggered when a big movement of a tag was detected                                                                                                                              |
-|                  | TAG_SMALL_MOVE     | Triggered when a small movement of a tag was detected                                                                                                                            |
-|                  | TAG_UNINSTALLED    | Triggered when a tag gets uninstalled                                                                                                                                            |
-|                  | TAG_OPEN           | Triggered when an open event of a tag was detected                                                                                                                               |
+**Supported trigger channels for the Home, Presence and Doorbell thing:**
+
+| Channel Type ID | Options            | Description                                                         |
```

Comment:
Are the previous events TAG_* now irrelevant ?

Reply:
These aren't triggered. I compared the README with the channels.xml (search for "home-event") and the code and found out, that a few options which are documented within the README are never triggered. The README was outdated.

Maybe @clinique could check if that is a bug and fix that with another change. But there seems not to be an open issue so probably there isn't a user which is missing the TAG_* event options.